models/stg_google_play__stats_crashes_app_version.sql

```diff
@@ -0,0 +1,34 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_google_play__stats_crashes_app_version_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_google_play__stats_crashes_app_version_tmp')),
+                staging_columns=get_stats_crashes_app_version_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        date as date_day,
+        app_version_code,
+        package_name,
+        sum(daily_anrs) as anrs,
+        sum(daily_crashes) as crashes
+    from fields
+    group by 1,2,3 -- for grouping NULL app version codes together into one pile (this might not ever happen but just to be safe...)
+)
```

Comment:
Do you have a preference between this way vs a `case when` statement, where `if null` then assign some kind of value such as 'Not Available'? In my experience, `null` cases are weird across db and with various functions, this may be a more explicit way in doing this, but totally open to discussion!

Reply:
Additionally, I am a little confused by the comment you've left here. Are there multiple app_version codes and package names and therefore require the group by anyway?